packages/edge-login-ui-rn/src/components/themed/EdgeTextField.js

```diff
@@ -3,9 +3,24 @@
 import * as React from 'react'
 import { TextField } from 'react-native-material-textfield'
 
-import { type ThemeProps, withTheme } from '../services/ThemeContext.js'
+import { unpackEdges } from '../../util/edges.js'
+import {
+  type Theme,
+  type ThemeProps,
+  withTheme
+} from '../services/ThemeContext.js'
 
-class EdgeTextFieldComponent extends React.PureComponent<ThemeProps> {
+type Props = {
+  // The gap around the button. Takes 0-4 numbers (top, right, bottom, left),
+  // using the same logic as the web `margin` property. Defaults to 0.
+  marginRem?: number[] | number,
+
+  // The gap inside the button. Takes 0-4 numbers (top, right, bottom, left),
+  // using the same logic as the web `padding` property. Defaults to 0.5.
+  paddingRem?: number[] | number
```

Comment:
We don't need padding here, since there is no "inside" like there is on a button.